pkg/chart/chart.go

```diff
@@ -812,7 +815,13 @@ func (t *Testing) ValidateMaintainers(chart *Chart) error {
 	return nil
 }
 
-func (t *Testing) PrintPodDetailsAndLogs(namespace string, selector string) {
+func (t *Testing) PrintEventsPodDetailsAndLogs(namespace string, selector string) {
+	util.PrintDelimiterLine("=")
+
+	printDetails(namespace, "Events of namespace", ".", func(item string) error {
+		return t.kubectl.GetEvents(namespace)
```

Comment:
Is it possible to use a label selector for events?

Reply:
Yes, that's possible. I guess that might be better if someone uses a pre-defined namespace

Reply:
Actually, that won't quite work because the events won't have the labels of the chart resources.

Reply:
You could do something like this:

`kubectl get events --field-selector 'involvedObject.name=foo'`

But that could be tricky. Depending on resource naming we might miss something.

Reply:
😞that's too bad... as long as tons of existing events doesn't bog down performance or place unnecessary load on the api server this seems okay for either case though.

Reply:
I dont see an issue here.